cmd/state-installer/installer.go

```diff
@@ -45,6 +48,22 @@ func NewInstaller(cfg *config.Instance, out output.Outputer, payloadPath string,
 }
 
 func (i *Installer) Install() (rerr error) {
+	isAdmin, err := osutils.IsAdmin()
+	if err != nil {
+		return errs.Wrap(err, "Could not determine if running as Windows administrator")
+	}
+	if isAdmin {
+		prompter := prompt.New(!i.Params.nonInteractive, i.an)
+		defaultChoice := !prompter.IsInteractive() // default to no for interactive, yes for non-interactive
+		confirm, err := prompter.Confirm("", locale.T("installer_prompt_is_admin"), &defaultChoice)
```

Comment:
```suggestion
	if isAdmin && !i.Params.nonInteractive {
		prompter := prompt.New(true, i.an)
		confirm, err := prompter.Confirm("", locale.T("installer_prompt_is_admin"), ptr.To(false))
```

Nit: I find it easier to understand what's happening this way.